src/k2/cmd/nodepool/nodepool_main.cpp

```diff
@@ -47,7 +46,6 @@ int main(int argc, char** argv) {
     app.addApplet<k2::CollectionMetadataCache>();
     app.addApplet<k2::NodePoolMonitor>();
     app.addApplet<k2::AssignmentManager>();
-    app.addApplet<k2::PartitionManager>();
```

Comment:
Why removing this? Leave us no possibility to host multiple partition (of same collection) in one node in the future?